### DISCUSSION TITLE
Striking out cancelled events

### DISCUSSION THREAD

**Discussion by The-Laug** [tine-groupware/tine#60] [Ideas]
Hello! 
I just updated my system from Tine20 to Tine Groupware, Ellie (2023.11.10). So far it has been quite seamless, thanks for that!

On the new installation, whenever I cancel an event it doesn't strike the summary of the event out, as it normally did. This was a functionality on the old installation, and I think it was a good addition. 
Would love to see it again!
![image](https://github.com/tine-groupware/tine/assets/54498065/194f7e91-4986-4bbc-92ab-8cb3cfb2bc21)
![image](https://github.com/tine-groupware/tine/assets/54498065/52219c7f-bf15-4573-aec9-e0d0a5f60bb0)

Best Wishes
Lauge

**Comment by ignoreigor** [comment]
I agree, this would be a helpful re-enabled feature. If one change the events status from confirmed to cancelled, it'll be striked out, but if one cancel it's participation, it is only filtered. When removing the filter, it looks like a "normal" event, no matter which colorset you are changing.

**Comment by pschuele** [comment]
the (display/css) bug has been fixed with 633479f7b8141584b3f1ffac18788cf5ce24b158

**Reply by ignoreigor** [reply]
Hi,

> the (display/css) bug has been fixed with [633479f](https://github.com/tine-groupware/tine/commit/633479f7b8141584b3f1ffac18788cf5ce24b158)

I am not sure, if this hits the point. In weekly-2024.25.1 there wasn't a misspelled css class anymore. @The-Laug did not refer to the status of the appointment, but to the cancelled participation (or maybe he did - and my thoughts were different). This CSS class is not assigned there, but I didn't find any others during a quick check yesterday.

An cancelled participation is either filtered and not visible or, if removing the filter, is visible and has classes "cal-monthview-event cal-monthview-event-editgrant **cal-status-CONFIRMED**" in it's wrapper - so it can't be distinguished visually between confirmed and cancelled events.
